server/config/contactForm.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    CONCERT_MAIL: ['concert@kfet-insa.fr', 'technique@kfet-insa.fr'],
+    EVENT_MAIL: ['sg@kfet-insa.fr', 'president@kfet-insa.fr'],
+    LOST_MAIL: ['sg@kfet-insa.fr'],
+    WEBSITE_MAIL: ['webmaster@kfet-insa.fr'],
+    RECAPTCHA: {
+        SECRET: '6LfIS1gUAAAAAKFZgZm1ZOVhzEFDMrD8GcV2W2X2'
```

Comment:
How to store the secret for development env, prod env ? @Embraser01

Reply:
Like I said, with env variables. 
For devs, I don't really know, but It's not really important for now

Reply:
See slack